### DISCUSSION TITLE
Tilføj lektier fra Huskelisten til Google Tasks

### DISCUSSION THREAD

**Discussion by sorentorp** [scaarup/aula#103] [Ideas]
Da Aula og især deres widgets ikke tilbyder notifikationer fra Aulas app ville det være fedt hvis de lektier ens barn får via Huskelisten, kan blive brugt via HomeAssistant. Det kunne fx. være at smide det videre til Google Tasks med GTask modulet til HomeAssistant.

Jeg er ikke den store programmør og ChatGPT forstod ikke hvad jeg ville ;-)

Er det overhovedet muligt når data er i Markdown-format? Hvordan sikrer jeg at det Tasks løbende er opdateret/ikke opretter dubletter osv?

Alle input og guidening er velkommen

**Comment by scaarup** [comment]
Du kan måske automatisere på om denne her huskeliste attribute ændrer sig?

**Comment by sorentorp** [comment]
Det er noget jeg skal have gravet ned i - igen, jeg er ikke programmør - så det må blive en dag hvor der er meget sne ;-)

**Comment by scaarup** [comment]
Det skulle heller ikke være nødvendigt :)

Fx vil denne her trigge hver gang "ugeplan" attributten ændrer sig:
![image](https://user-images.githubusercontent.com/8055470/236223413-145dc43e-29dd-47c2-a8b9-6bed67c86e56.png)

Du burde kunne gøre det samme, blot på din huskeliste attribute.

**Comment by sorentorp** [comment]
Det bør være muligt, ja. Jeg får kigget på det :-)
